postgres-appliance/Dockerfile

```diff
@@ -50,6 +51,9 @@ RUN export DEBIAN_FRONTEND=noninteractive \
             /usr/share/locale/??_?? \
     && find /var/log -type f -exec truncate --size 0 {} \;
 
+RUN  apt-get install curl \
+        && curl -sL https://aka.ms/InstallAzureCLIDeb | bash
```

Comment:
According to the wal-e documentation we don't need it.
There is also nothing worse than piping into bash...

Reply:
Eliminated this code